dbfit-java/derby/src/main/java/dbfit/environment/DerbyEnvironment.java

```diff
@@ -116,16 +116,16 @@ private Direction getParamDirection(int columnType, boolean isFunction) {
                       + (isFunction ? "function" : "procedure") + " parameters");
     }
 
-    private DbParameterAccessor crtProcDbParamAccessor(ResultSet rs,
-                                                       boolean isFunction)
-                                                       throws SQLException {
-        String paramName = defaultIfNull(rs.getString("COLUMN_NAME"), "");
+    private DbParameterAccessor createObjectDbParamAccessor(String paramName,
```

Comment:
Some doubts about the name, do we need the `Object` part (gets me thinking whether we have accessors to non-objects).